browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -246,3 +267,66 @@ bool BraveTabContextMenuContents::IsValidContextMenu() const {
 void BraveTabContextMenuContents::OnMenuClosed() {
   menu_closed_ = true;
 }
+
+void BraveTabContextMenuContents::NewSplitView() {
+  auto* model = browser_->tab_strip_model();
```

Comment:
The function `NewSplitViewForActiveTab` is very similar to this one - maybe it could optionally specify a tab?

Reply:
Good call!